### ISSUE TITLE
Identical nodes in connections tree

### ISSUE THREAD

**Issue by Googleboom** [mRemoteNG/mRemoteNG#2414]
I have several servers that are shared between tree branches (projects). Nodes have the same server name/IP and same user name+domain. 
When I have a RDP connection already opened in one branch and click on the identical node in another branch, I am asked to enter credentials and the connection is re-opened. 

## Expected Behavior

Existing tab should be opened instead. A node in the connection tree should have "connected" icon when opened in another branch.

## Current Behavior

A new connection is opened, the old tab is closed (RDP connection terminates existing session)

**Comment by BlueBlock** [comment]
That would make sense to just go to the open tab but only if the username is different.

**Comment by Googleboom** [comment]
> That would make sense to just go to the open tab but only if the username is different.

No, especially if the user is the same. A reconnection is mandatory anyway if users are different.
Just as an example of the case:
10 projects/branches. Each contains a Live server and a SharedResource server.
The SharedResource server has a single isntance and is shared for all (or only part) projects.

I work with Project1 and have the SharedResource server connected.
Then I need to open the SharedResource server for Project10.

It is already opened in existing tab.
I want to open this tab when I click Project10/SharedResource node.
I might not remember that it is already open in connection with another project or might not knowing that it is shared at all.

**Comment by BlueBlock** [comment]
I think we agree. Correct me if I'm wrong.

I incorrectly explained what I was trying to say:
That would make sense to just go to the open tab but only if the username is -the same-.

**Comment by robertpopa22** [comment]
**Fix available for testing**

**Commit:** `fe43053f` (fork `robertpopa22/mRemoteNG`, branch `main`)
**What changed:** Identical nodes in connections tree

A beta build (v1.81.0-beta.4) including this fix is available from the fork's Releases page.

---

> *We sincerely appreciate the mRemoteNG maintainers and the wonderful community that has kept this project alive and thriving for over a decade. Your work powers thousands of IT professionals worldwide, including us. Our fork builds entirely on your foundation — we're preparing a consolidated Pull Request to contribute these improvements back to the official project. Thank you for everything you've built.*
>
> *— robertpopa22, community contributor*
